package.json

```diff
@@ -43,43 +75,80 @@
     "singleQuote": true
   },
   "devDependencies": {
-    "@babel/cli": "^7.2.3",
-    "@babel/core": "^7.2.2",
-    "@babel/plugin-transform-runtime": "^7.3.4",
-    "@babel/polyfill": "^7.2.5",
-    "@babel/preset-env": "^7.3.1",
-    "@babel/preset-react": "^7.23.3",
+    "@babel/plugin-transform-runtime": "^7.24.0",
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
     "@mui/material": "^5.8.0",
-    "@storybook/addon-actions": "^7.6.7",
-    "@storybook/addon-interactions": "^7.6.7",
+    "@storybook/addon-essentials": "8.0.0",
+    "@storybook/addon-interactions": "^8.0.0",
+    "@storybook/addon-links": "^8.0.0",
     "@storybook/api": "^7.6.7",
-    "@storybook/jest": "^0.2.3",
-    "@storybook/manager-api": "^7.6.7",
-    "@storybook/preview-api": "^7.6.7",
-    "@storybook/react": "^7.6.7",
-    "@storybook/react-webpack5": "^7.6.7",
-    "@storybook/testing-library": "^0.2.2",
-    "babel-loader": "^8.0.5",
+    "@storybook/blocks": "^8.0.0",
+    "@storybook/components": "^8.0.0",
+    "@storybook/core-events": "^8.0.0",
+    "@storybook/manager": "^8.0.0",
+    "@storybook/manager-api": "^8.0.0",
+    "@storybook/preview": "^8.0.0",
+    "@storybook/preview-api": "^8.0.0",
+    "@storybook/react": "^8.0.0",
+    "@storybook/react-vite": "^8.0.0",
+    "@storybook/test": "^8.0.0",
+    "@storybook/theming": "^8.0.0",
+    "@storybook/types": "^8.0.0",
+    "@types/archiver": "^6.0.2",
+    "@types/babel__plugin-transform-runtime": "^7",
+    "@types/node": "^18.15.0",
+    "@types/react": "^18.2.55",
+    "@types/react-dom": "^18.2.19",
+    "@types/styled-components": "^5.1.34",
+    "@vitejs/plugin-react": "^4.2.1",
+    "archiver": "^3.0.0",
+    "auto": "^11.0.4",
+    "boxen": "^7.1.1",
+    "dedent": "^1.5.1",
     "eslint": "^8.6.0",
     "eslint-plugin-jest": "^27.6.3",
     "eslint-plugin-react": "^7.28.0",
-    "file-loader": "^3.0.1",
     "happo.io": "^9.0.0",
     "jest": "^29.7.0",
-    "raw-loader": "^1.0.0",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^3.2.5",
+    "prompts": "^2.4.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-tether": "^2.0.0",
-    "storybook": "^7.6.7",
+    "rimraf": "^5.0.5",
+    "storybook": "^8.0.0",
     "styled-components": "^4.1.3",
-    "webpack": "^4.0.0"
+    "tsup": "^8.0.2",
+    "typescript": "^5.3.3",
+    "vite": "^5.1.2",
+    "zx": "^7.2.3"
   },
-  "dependencies": {
-    "@babel/runtime": ">=7.0.0",
-    "archiver": "^3.0.0",
-    "rimraf": "^2.6.3"
+  "packageManager": "yarn@4.0.2",
+  "storybook": {
+    "displayName": "Happo",
+    "supportedFrameworks": [
+      "supported-frameworks"
```

Comment:
This looks like a placeholder. I think I'm going to try removing `supportedFrameworks` altogether in the hopes that this will signal that Happo supports any Storybook framework. Which in theory it should.